packages/react-dom/src/events/ReactDOMEventListener.js

```diff
@@ -159,11 +160,15 @@ export function addTrappedEventListener(
   if (passive === true && !passiveBrowserEventsSupported) {
     passive = false;
   }
+  const eventSystemFlags =
+    enableLegacyFBSupport && legacyFBSupport
```

Comment:
this is a bit confusing because you'd imagine legacyFBSupport would always be true if enableLegacyFBSupport is true. I think it means this is a badly named argument. Something to consider for a future PR?

Reply:
I'll rename as part of this PR, good point though.